bundles/core/src/main/java/com/adobe/cq/wcm/core/components/sandbox/internal/models/AbstractContainerImpl.java

```diff
@@ -39,7 +50,15 @@
     @Self
     private SlingHttpServletRequest request;
 
+    @Inject
+    private ModelFactory modelFactory;
+
+    @Inject
+    private SlingModelFilter slingModelFilter;
+
     private List<ListItem> items;
+    private Map<String, ? extends ComponentExporter> childrenModels;
```

Comment:
would prefer `childModels` here

Reply:
I've moved this change to issue/298 branch and it will be merged from a new PR